robosat/datasets.py

```diff
@@ -42,21 +42,27 @@ def __getitem__(self, i):
 # Multiple Slippy Map directories.
 # Think: one with images, one with masks, one with rasterized traces.
 class SlippyMapTilesConcatenation(torch.utils.data.Dataset):
-    '''Dataset to concate multiple input images stored in slippy map format.
-    '''
+    """Dataset to concate multiple input images stored in slippy map format.
+    """
 
     def __init__(self, inputs, input_transforms, target, target_transform):
         super().__init__()
 
         # No-op transform needs to be expressed with identify function `id`
-        assert len(inputs) == len(input_transforms), 'one transform per input directory'
-        assert len(inputs) > 0, 'at least one input slippy map dataset to compose'
+        assert len(inputs) == len(input_transforms), "one transform per input directory"
+        assert len(inputs) > 0, "at least one input slippy map dataset to compose"
 
-        self.inputs = [SlippyMapTiles(inp, fn) for inp, fn in zip(inputs, input_transforms)]
+        self.inputs = [
+            SlippyMapTiles(inp, fn) for inp, fn in zip(inputs, input_transforms)
+        ]
         self.target = SlippyMapTiles(target, target_transform)
 
-        assert len(set([len(dataset) for dataset in self.inputs])) == 1, 'same number of tiles in all inputs'
-        assert len(self.target) == len(self.inputs[0]), 'same number of tiles in inputs and target'
+        assert (
+            len(set([len(dataset) for dataset in self.inputs])) == 1
+        ), "same number of tiles in all inputs"
+        assert len(self.target) == len(
+            self.inputs[0]
+        ), "same number of tiles in inputs and target"
```

Comment:
We should at least adapt the line length to what we specify in the flake config. Or why is it breaking here?

Reply:
Looks like was from the `--line-length` parameter. Now, set that to 120 and code is formatted as we expect and looks good as well. 😄